src/main/java/org/folio/processing/mapping/mapper/writer/marc/MarcRecordModifier.java

```diff
@@ -146,6 +141,24 @@ public void processUpdateMappingOption(List<MarcMappingDetail> marcMappingRules)
     }
   }
 
+  public String updateRecord(Record srcRecord, Record recordToUpdate, List<MarcFieldProtectionSetting> protectionSettings) {
```

Comment:
can you add a comment to this method describing what it does with any quirks, gotchas it might have?

Reply:
added comment